pkg/azureclient/client.go

```diff
@@ -25,10 +25,10 @@ type Client interface {
 	CreateOrUpdateZone(ctx context.Context, resourceGroupName string, zone string) (dns.Zone, error)
 	DeleteZone(ctx context.Context, resourceGroupName string, zone string) error
 	GetZone(ctx context.Context, resourceGroupName string, zone string) (dns.Zone, error)
-	ListZones(ctx context.Context, resourceGroupName string) (*[]dns.Zone, error)
+	ListZones(ctx context.Context, resourceGroupName string, top *int32) (dns.ZoneListResultPage, error)
```

Comment:
This function is not used anywhere.

Reply:
Yes. I realized that I don't need it in Azure since there isn't a different name for zones and I'm not using Status field for Azure. I will remove it

Reply:
Removed